server/jetty/src/main/java/io/deephaven/server/jetty/JsPlugins.java

```diff
@@ -1,40 +1,65 @@
 package io.deephaven.server.jetty;
 
+import io.deephaven.configuration.ConfigDir;
 import io.deephaven.configuration.Configuration;
 import org.eclipse.jetty.security.ConstraintSecurityHandler;
 import org.eclipse.jetty.server.Handler;
 import org.eclipse.jetty.servlet.DefaultServlet;
 import org.eclipse.jetty.servlet.ErrorPageErrorHandler;
+import org.eclipse.jetty.util.resource.PathResource;
 import org.eclipse.jetty.util.resource.Resource;
 import org.eclipse.jetty.webapp.WebAppContext;
+import org.jetbrains.annotations.NotNull;
 
 import java.io.IOException;
-import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Optional;
 import java.util.function.Consumer;
 
 import static org.eclipse.jetty.servlet.ServletContextHandler.NO_SESSIONS;
 
 class JsPlugins {
 
     public static void maybeAdd(Consumer<Handler> addHandler) {
+        resource()
```

Comment:
Can we log what directory we're reading from? If I'm running `./gradlew server-jetty-app:run`, I really don't know where this default directory is. Some logging would at least help debugging, or give me a clue to where it's looking.

Less of a concern when running from a `Dockerfile`

Reply:
Do you think logging in the no-property-set, no-default-directory-exists, is useful? I think logging that "no property has been set, no js-plugins directory existst here" is a bit extraneous (I imagine if we revamp our configuration framework, this might be a _general_ feature for we'd like for debugging any property).

I think logging the positive is completely fine. Will add this to this PR.

https://deephaven.io/core/groovy/docs/how-to-guides/configuration/js-plugins/ will be updated when this PR lands, describing that JS Plugins inherits the default directory of `<configDir>/js-plugins/`.

The configuration directory depends what system you are running on, see https://deephaven.io/core/docs/how-to-guides/configuration/native-application/#config-directory

Reply:
I think it would be useful if the directory doesn't exist so you know what it's looking for. I'm still not certain where the "default" config directory is.

Reply:
> The config directory is based on OS-conventions by default. On Linux, this is `$HOME/.config/<application>/`, and on macOS this is `$HOME/Library/Application Support/io.Deephaven-Data-Labs.<application>/`.